lib/frontend/get-github-configuration.js

```diff
@@ -45,6 +45,7 @@ module.exports = async (req, res, next) => {
         info,
         jiraHost,
         clientKey,
+        APP_URL: process.env.APP_URL,
```

Comment:
should this be app_url ? other props are in small cases

Reply:
done